src/PostgREST/Types.hs

```diff
@@ -163,7 +163,13 @@ operators = M.fromList [
   ("is", "IS"),
   ("@@", "@@"),
   ("@>", "@>"),
-  ("<@", "<@")]
+  ("<@", "<@"),
+  ("&&", "&&"),
```

Comment:
Good point, in that case I'm fine with what you suggested. If anyone else needs to weigh in on this, add them to the discussion, otherwise I'll update the PR.